services/edit/cache.js

```diff
@@ -148,4 +174,5 @@ exports.getSchema = control.memoizePromise(getSchema);
 
 // forgets
 exports.saveThrough = saveThrough;
+exports.saveForHTML = saveForHTML;
 exports.createThrough = createThrough;
```

Comment:
eventually (when we have a "create but gimme html plz" endpoint), we could also have a `createForHTML` function